modules/Testing-Snapshot--Goal-446/README.md

```diff
@@ -42,7 +42,7 @@ You will start by cloning a starter app the has full CRUD functionality. From he
   - A test for the `/contacts/new` GET route that checks that the correct page is rendered
   - A test for the `/contacts` POST route that saves contact data to the database
   - A test for the `/contacts/:contactId` GET route that makes sure the correct data is returned
-  - A test for the `/contacts/:contactId/delete` GET route that makes sure the correct data is deleted
```

Comment:
I had it this way despite its deviation from proper restful routes because its easier to create a form that does a post than a form that does a `delete` method request. Otherwise you'll have to use JavaScript to create a delete link.